paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentElement.kt

```diff
@@ -134,7 +135,7 @@ private fun FormElement(
                 }
             }
     ) {
-        if (selectedItem.code == PaymentMethod.Type.USBankAccount.code) {
+        if (selectedItem.code == USBankAccount.code || selectedItem.code == Link.code) {
```

Comment:
For now, let’s use the same messy integration like US Bank Account. We can work with the Payments team to refactor this at a later time.

Reply:
Is the idea to entirely reuse `USBankAccountForm` and its viewmodel?

Reply:
Yep, that’s the plan!